ambry-tools/src/main/java/com.github.ambry/tools/perf/ServerWritePerformance.java

```diff
@@ -320,6 +322,10 @@ public ServerWritePerfRun(int threadIndex, Throttler throttler, AtomicBoolean is
       this.measurementIntervalNs = measurementIntervalNs;
       this.enableVerboseLogging = enableVerboseLogging;
       this.connectionPool = connectionPool;
+      Properties props = new Properties();
+      props.setProperty("router.hostname", "localhost");
+      props.setProperty("router.datacenter.name", "localDC");
+      blobIdVersion = new RouterConfig(new VerifiableProperties(props)).routerBlobIdCurrentVersion;
```

Comment:
I wonder if the default could just be stored as a constant inside of `RouterConfig`, instead of having to construct properties objects in all of these places.

Reply:
This change is not very extensive (only one place in the production code creates a `BlobId`). Rest of the changes are in tests (for which we created a method in `CommonTestUtils`) and these are the tools.

There is a problem with keeping it as a constant in `RouterConfig` (basically consumers of it are interested in actuals not defaults). We could potentially use a `static` method, but then there are initialization order problems to worry about.